velox/functions/lib/Re2Functions.h

```diff
@@ -39,11 +39,23 @@ enum class PatternKind {
   kPrefix,
   /// Fixed pattern preceded by one or more '%', such as '%foo', '%%%hello'.
   kSuffix,
+  /// Patterns matching '%{c0}%', such as '%foo%%', '%%%hello%'.
+  kFixedCenter,
```

Comment:
Wondering if kSubstring is a better name? This pattern just says that a substring is found inside a string, no?

CC: @pramodsatya @laithsakka

Reply:
Makes sense, updated the name, thanks for the suggestion.

Reply:
I do not have strong opinion @pramodsatya